telegram/_business.py

```diff
@@ -246,7 +246,7 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["BusinessIntr
 
 class BusinessLocation(TelegramObject):
     """
-    This object represents the location of a business account.
+    This object Contains information about the location of a Telegram Business account.
```

Comment:
```suggestion
    This object contains information about the location of a Telegram Business account.
```
Notification falsely says "requested your review", so found this :)